Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -83,10 +83,8 @@ PYTHON_V="python3"  # starting from ubuntu 20.04, python isn't symlink to defaul
 PIP=pip3
 
 if [ ${RELEASE_CODENAME} == 'bionic' ] ; then
-    SITLFML_VERSION="2.4"
-    SITLCFML_VERSION="2.4"
-    PYTHON_V="python3"
-    PIP=pip3
+    echo "ArduPilot no longer supports developing on this deprecated operating system."
```

Comment:
```suggestion
    echo "ArduPilot no longer supports developing on this operating system that has reached end of standard support."
```

Reply:
![image](https://github.com/user-attachments/assets/f06c6641-8e65-4ca6-b96f-b1229ed0d2eb)

This matches the official language. Thoughts?

Reply:
Seems reasonable enough.